specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimanagement.json

```diff
@@ -542,6 +542,21 @@
       },
       "x-ms-parameter-location": "method"
     },
+    "PolicyFragmentIdParameter": {
+      "name": "policyFragmentId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "policyFragment"
+      ],
+      "description": "The identifier of the policy fragment.",
+      "x-ms-enum": {
```

Comment:
If this is an ID, surely it is just a string, not an enum?

Reply:
I'd ask the same question about PolicyIdParameter which is defined above too, except it seems to define and only have a single value. Is the PolicyIdParameter definition really an ID (object identifier)... or is it really intended more as a PolicyIdKind/PolicyIdType?

Reply:
Honestly, it's my first time doing this and I looked at the Policies API for guidance on how to do this. That's why this ID parameter is an enum in both.

I'll ask a team member why it's an enum for Policies and maybe we'll have to fix it in both.

Reply:
I fixed it for fragments. The parameter is now a string.

For Policies, having it as an enum is a way to comply with arm requirements:

> arm requires every resource to be modeled as /collection/:id.
But each entity that has policy in APIM (service, api, operation, product) can have only one policy. So if not arm restriction we would do smth like /apis/:id/policy. But to comply we have to do /api/:id/policies/policy